src/scripts/main.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+// for (let i = 1;)
+
+// .stars--1 :nth-child(-n+1),
+// .stars--2 :nth-child(-n+2),
+// .stars--3 :nth-child(-n+3),
+// .stars--4 :nth-child(-n+4),
+// .stars--5 :nth-child(-n+5) {
+//   background-image: url(../images/star-active.svg);
+// }
```

Comment:
are these your changes? please delete this

Reply:
Yes, that's mine. I was thinking of implementing the star block via a javascript loop.
Mentor suggested this:
[loralevitska](https://github.com/loralevitska) [last month](https://github.com/mate-academy/layout_product-cards/pull/2594#discussion_r1112133224)
Try to rewrite it by loop for more practice. If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review.